rust_src/src/lisp.rs

```diff
@@ -20,13 +20,15 @@ use crate::{
     lists::{list, memq, CarIter, LispCons, LispConsCircularChecks, LispConsEndChecks},
     multibyte::LispStringRef,
     process::LispProcessRef,
+    remacs_sys::specbind_tag::*,
```

Comment:
Why not import `specbind_tag` and use it `specbind_tag::SPECPDL_LET`?

`*` imports have a way of biting you at some point.

Reply:
I don't think that there will be another `SPECPDL_LET` but agreed, better to be sure. Fixed

Reply:
I still see the '*' import here.

Reply:
Whoops, my bad. Forgot to stage file. Fixed.